README.md

```diff
@@ -106,5 +108,129 @@ If volumes don't mount because of selinux, this command may be your friend:
   ```chcon -Rt svirt_sandbox_file_t /path/of/volume```
 
 Take a look at run_gaffer, a script which starts the four containers.
-Also, kubernetes/README.kubernetes.md if you want to run Gaffer in Kubernetes.
+Also, <kubernetes/README.kubernetes.md> if you want to run Gaffer in Kubernetes.
+
+To set up a cluster, you need to take control of address allocation.
+
+```
+  ############################################################################
```

Comment:
Would it be possible for some of this network configuration to be set up in the run_gaffer script? Using something like getopts to provide some network information (maybe a CIDR range and port range), then using that information to start the instances might make it a little easier to get up and running with a small cluster.